crates/trie/trie/src/updates.rs

```diff
@@ -449,3 +448,217 @@ mod tests {
         assert_eq!(updates_deserialized, default_updates);
     }
 }
+
+/// Bincode-compatible trie updates type serde implementations.
+#[cfg(all(feature = "serde", feature = "serde-bincode-compat"))]
+pub mod serde_bincode_compat {
```

Comment:
please move this above the test mod, having this below is weird

Reply:
yeah fair, i started this way and then didn't want to introduce different ways 😄 will also create a good first issue to fix all of them at once